.github/workflows/e2e.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
           flux create source git flux-system \
           --url=${{ github.event.repository.html_url }} \
           --branch=${GITHUB_REF#refs/heads/} \
-          --ignore-paths="./clusters/**/flux-system/"
```

Comment:
The asteriks (`**`) should work fine, and should be there for the testing purpose as well. Why replace it with `staging`?
Consider only removing the faulty dot (`.`) from the beginning of the string.

Reply:
I replaced it with `staging` as I was not sure that the issue was with `.` or with `**`. I will remove the dot and add back the glob style pattern and test it in my repo